README-usage.md

```diff
@@ -2,22 +2,34 @@
 
 ## ..MPyL CLI
 
-Install MPyL
+#### 1. Install MPyL
 
 ```shell
 pip install mpyl
 mpyl --help
 ```
 
-#### Health check
+#### 2. Health check
 
 ⭐ It is recommended to run this before running any other commands.
 ```shell
 mpyl health
 ```
 Will validate the configuration and check if all required tools are installed.
 
-##### Command structure
+#### 3. Create a PR with your changes
+
+```shell
+gh pr create --draft
```

Comment:
Do we mention the gh cli requirement anywhere?

Reply:
It's mentioned in mpyl health

Reply:
Perhaps we could mention somewhere that this a _recommended_ workflow.